frontend/client/components/Profile/index.tsx

```diff
@@ -176,26 +203,23 @@ class Profile extends React.Component<Props, State> {
                 ))}
               </div>
             </Tabs.TabPane>
-            <Tabs.TabPane tab={TabTitle('Funded', contributions.length)} key="funded">
-              <div>
-                {noneFunded && (
-                  <Placeholder
-                    loading={isLoading}
-                    title="No proposals funded"
-                    subtitle="There have not been any proposals funded."
-                  />
-                )}
-                {contributions.map(c => (
-                  <ProfileContribution
-                    key={c.id}
-                    userId={user.userid}
-                    contribution={c}
-                    showSendInstructions={this.openContributionModal}
-                  />
-                ))}
-              </div>
-            </Tabs.TabPane>
+            {!noneFunded && (
+              <Tabs.TabPane tab={TabTitle('Funded', contributions.length)} key="funded">
+                {renderTabMsg(publicMsg)}
```

Comment:
We're using the public message for the `Funded` tab, though technically funded proposals can be either public or private. Should we display a custom note instead?